pymatviz/ptable/_projector.py

```diff
@@ -552,8 +550,8 @@ def add_annotation(
                 symbol_color = text_color
 
             ax.text(
-                *pos,
-                text.get(symbol, ""),
+                *kwargs.pop("pos"),
```

Comment:
As this operates inside a loop, I guess this would only get the desired value for the first iteration?